apps/vendors/views.py

```
@@ -0,0 +1,74 @@
+    def verify(self, request, pk=None):
+        """Mark a vendor as verified (staff only)."""
+
+        vendor = self.get_object()
+        if not (request.user.is_staff or request.user.is_superuser):
+            return Response({'detail': 'Permission denied.'}, status=403)
+        verify_vendor(vendor)
+        serializer = self.get_serializer(vendor)
+        return Response(serializer.data)
+
+    @action(detail=True, methods=['post'])
+    def deactivate(self, request, pk=None):
```

Ruff (0.14.0)

Unused method argument: `pk`

(ARG002)